controller/controller_test.go

```diff
@@ -6586,6 +6586,7 @@ containers:
 		It("should return error when it fails to create pod", func() {
```

Comment:
Do you think it is worth to test both scenarios?
1. When only one worker has to create both pods;
2. When two or more workers take pods to create (what this current version tests);

Reply:
I think it makes sense. I will add another test.